test/mocks/handlers/tripReq.handler.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable import/prefer-default-export */
+import { rest } from 'msw';
+// import baseURl from '../../../src/api';
+
+export const tripReqHandlers = [
+  rest.get('http://localhost:5000/api/v1/trip', (req, res, ctx) => {
+    if (!res.data) {
+      return res(
+        ctx.status(400),
+        ctx.json({
+          Error: 'Error',
+          status: 400,
+        }),
+        ctx.delay(100)
+      );
+    }
```

Comment:
The res object is empty, you are supposed to use it to send a mock response to the server because there won't be actual server call.